apps/jobs/views.py

```diff
@@ -534,6 +535,45 @@ def get_jobs(self, request):
         return response.create_response(
             serialized_filtered_jobs_data, status.HTTP_200_OK
         )
+    
+    @action(detail=False, methods=['get'])
+    def get_jobs_categories(self, request):
+        """
+        API: /get_jobs_categories
+        Return open positions present in specific job category
+        """
+        
+        try:
+            job_data = self.queryset.filter(is_created=True, is_deleted=False)
+            
+            if job_data.exists():
+                category_counts = {}
+
+                for job in job_data:
+                    category = job.category.lower().strip()
+                    category_counts[category] = category_counts.get(category, 0) + 1
+
+                open_positions_in_category = [
+                    {"id": str(uuid.uuid4()), "category": category, "open_position": count}
```

Comment:
you can send integer id its fine no need of uuid

Reply:
Updated Response:
```
{
    "data": [
        {
            "id": 1,
            "category": "marketing",
            "open_position": 2
        },
        {
            "id": 2,
            "category": "tech",
            "open_position": 4
        },
        {
            "id": 3,
            "category": "science",
            "open_position": 2
        }
    ]
}
```